dags/probe_scraper.py

```diff
@@ -125,4 +125,30 @@
         }
     )
 
-    delay_python_task >> lookml_generator
+    delay_python_task >> lookml_generator_prod
+
+    lookml_generator_staging = GKEPodOperator(
+        email=["frank@mozilla.com", "dataops+alerts@mozilla.com"],
+        task_id="lookml_generator_staging",
+        name="lookml-generator-staging-1",
+        image="gcr.io/moz-fx-data-airflow-prod-88e0/lookml-generator:latest",
```

Comment:
staging is using latest, prod is using release?

Reply:
That's the plan